setup.py

```diff
@@ -1,13 +1,14 @@
 from distutils.core import setup
 
 setup(
-    name = 'ocrspace',
-    packages = ['ocrspace'], # this must be the same as the name above
-    version = '2.3.0',
-    description = 'Perform OCR through ocr.space API',
-    author = ['Ali Najafi', 'Erik Boesen'],
-    author_email = 'me@erikboesen.com',
-    url = 'https://github.com/ErikBoesen/ocrspace',
-    keywords = ['ocr'],
-    classifiers = [],
+    name='ocrspace',
+    packages=['ocrspace'],  # this must be the same as the name above
+    requires=['requests'],
+    version='2.3.0',
+    description='Perform OCR through ocr.space API',
+    author=['Ali Najafi', 'Erik Boesen', 'Negassa Berhanu'],
+    author_email='forpurposes1435@gmail.com',
```

Comment:
Please leave my email in there

Reply:
resolved on c9ffd78ddc93ede49af145fe706ee269c4e0a42c...with my apologies